docs/sources/installation/upgrading.md

```diff
@@ -58,29 +58,44 @@ restore:
 > psql grafana < grafana_backup
 ```
 
-### Ubuntu / Debian
+### Ubuntu or Debian
 
-If you installed grafana by downloading a debian package (`.deb`) you can just follow the same installation guide
-and execute the same `dpkg -i` command but with the new package. It will upgrade your Grafana install.
+You upgrade Grafana by following the same procedure as when you installed it.
 
-If you used our APT repository:
+#### Upgrade Debian package
+
+If you installed Grafana by downloading a Debian package (`.deb`), then you can execute the same `dpkg -i` command but with the new package. It will upgrade your Grafana installation.
+
+Go to the [download page](https://grafana.com/grafana/download?platform=linux) for the latest download
+links.
+
+```bash
+wget <debian package url>
+sudo apt-get install -y adduser libfontconfig1
+sudo dpkg -i grafana_<version>_amd64.deb
+```
+
+#### Upgrade from APT repository
+
+If you installed Grafana from our APT repository:
```

Comment:
We should probably note here that if you use the APT repository then grafana will automatically be updated when you run `apt-get upgrade` to upgrade all system packages.

Reply:
Done